roles/neutron-common/templates/usr/local/bin/neutron-reset

```diff
@@ -61,16 +64,14 @@ class Interface(object):
             return True
 
         # veth interfaces named tap... belong to Neutron
-        if re.match(r'^tap[0-9a-z]{8}-[0-9a-z]{2}$', self.ifname) and \
-                self.iftype() == 'veth':
-            return True
+        if re.match(r'^tap[0-9a-z]{8}-[0-9a-z]{2}$', self.ifname):
+           if self.iftype() == 'tun' or self.iftype() == 'veth':
```

Comment:
Interfaces created by QEMU on behalf of libvirt and nova-compute will match this condition, so this function will now incorrectly return true for interfaces which were not created by Neutron.